src/formatDate.js

```diff
@@ -8,7 +8,57 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitElement = fromFormat[3];
+  const joinElement = toFormat[3];
+  const newDate = date.split(splitElement);
+
+  let formatAgan = [];
```

Comment:
The variable name `formatAgan` is a bit unclear. Consider renaming it to something more descriptive, such as `dateComponents` to better reflect its purpose.